config/initializers/email.rb

```diff
@@ -1,18 +1,22 @@
-SearchGovInterceptor = Struct.new(:force_to) do
-  def delivering_email(message)
-    message.to = [force_to]
+# frozen_string_literal: true
+
+Rails.application.config.to_prepare do
```

Comment:
The diff looks much bigger than it is due to the change in indentation. The only relevant change is that the code of this initializer is now wrapped in a `to_prepare` block:
```ruby
Rails.application.config.to_prepare do
  # pre-existing code
end
```
This change is required by Rails 7, and was not surfaced while testing because the staging stack used for testing did not specify a `force_to` address. See https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#use-case-1-during-boot-load-reloadable-code for info on the fix.

Reply:
Thanks for the ref.